bin/attach

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker attach $(docker ps -f name=psulib_blacklight_web_1 -q)
```

Comment:
this might have snuck in there for some other reason

Reply:
I added this on purpose, find it handy to attach to the running container with just `bin/attach` for when debugging with pry or so.

Reply:
I think `docker-compose web` will attach to the current web container. But I'm curious why you would need that for debugging. Are you running your tests inside the container?

Reply:
Correct, it attaches to the current so for debugging. And yes I am running tests inside the container.

Reply:
Running the tests outside of docker is much faster, in my experience. The only thing I use docker for is Solr. I run redis locally, although you don't have to. We should talk maybe? I can show you how I have it set up. I wrote up an updated dev setup here: https://github.com/psu-libraries/psulib_blacklight/wiki/Development-Setup

Reply:
right, it is a bit slower. I will try to go with the way instructions are and see, thanks 👍🏻

Reply:
Sure thing. Please bug me if you have problems.

Reply:
sure, thanks.

I also removed this.